hw/bsp/imxrt/family.mk

```diff
@@ -8,13 +8,18 @@ CPU_CORE ?= cortex-m7
 MCU_VARIANT_WITH_CORE = ${MCU_VARIANT}${MCU_CORE}
 MCU_DIR = $(SDK_DIR)/devices/$(MCU_VARIANT)
 
+CFLAGS += \
+  -D__STARTUP_CLEAR_BSS \
+  -DCFG_TUSB_MCU=OPT_MCU_MIMXRT1XXX \
+  -DCFG_TUSB_MEM_SECTION='__attribute__((section("NonCacheable")))' \
```

Comment:
On M7 core `NonCacheable` is located on DTCM so there is no need to add a if switch.